src/sqs/deployment.ts

```diff
@@ -0,0 +1,9 @@
+import { WebhookPayloadDeploymentStatus } from "@octokit/webhooks";
+
+export type DeploymentMessagePayload = {
+	jiraHost: string,
+
+	// The original webhook payload from GitHub. We don't need to worry about the SQS size limit because metrics show
+	// that payload size for deployment_status webhooks maxes out at 13KB.
+	webhookPayload: WebhookPayloadDeploymentStatus
+}
```

Comment:
The SQS message payload will just contain the plain webhook payload since it's small enough.